.github/workflows/auto-public-pr-copybara.yml

```
@@ -0,0 +1,90 @@
+name: Copybara - Auto Public PR Migration
+
+on:
+  pull_request_target:
+    types: [opened, reopened, synchronize, labeled, unlabeled]
+    paths-ignore:
+      - '.github/**'
+      - '**/dbt-parser/src/resolver.rs'
```

```suggestion
  pull_request:
```

There's no point in excluding the one resolver.rs (and we definitely want to run the tests when changing github workflow itself)